drivers/ws281x/esp32.c

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Christian Friedrich Coors
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ws281x
+ *
+ * @{
+ *
+ * @file
+ * @brief       Implementation of `ws281x_write_buffer()` for the ESP32 CPU
+ *
+ * @author      Christian Friedrich Coors <me@ccoors.de>
+ *
+ * @}
+ */
+#include <assert.h>
+#include <errno.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "ws281x.h"
+#include "ws281x_params.h"
+#include "ws281x_constants.h"
+#include "periph_cpu.h"
+#include "xtimer.h"
+#include "xtensa/core-macros.h"
+#include "soc/rtc.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define NANOSECOND          (0.000000001)
+
+static inline __attribute__((always_inline)) uint32_t getCycleCount(void) {
```

Comment:
According to the coding convention, this should be named `get_cycle_count()` (No CamelCase.)